app/screens/send-bitcoin-screen/send-bitcoin-reducer.ts

```diff
@@ -1,151 +1,142 @@
 import { Destination, InvalidDestination } from "./payment-destination/index.types"
 
-export const DestinationState = {
-  Entering: "entering",
-  Validating: "validating",
-  Valid: "valid",
-  RequiresUsernameConfirmation: "requires-destination-confirmation",
-  Invalid: "invalid",
-} as const
-
-export type DestinationState = (typeof DestinationState)[keyof typeof DestinationState]
+export enum DestinationState {
```

Comment:
we don't use `enum` in our codebase, it has some flaws for typescript IIRC / don't remember what exactly.

instead we use const object,with the `as const` to tell typescript that even the properties are immutable, ie: 

```
export const DestinationState = {
  Entering: "entering",
  Validating: "validating",
  Valid: "valid",
  RequiresUsernameConfirmation: "requires-destination-confirmation",
  Invalid: "invalid",
} as const
```

Reply:
Reverted the enums to how they were previously